src/GovUk.Education.ExploreEducationStatistics.Data.Model/Release.cs

```diff
@@ -22,7 +22,6 @@ public class Release
         // TODO by the PublishReleaseContentFunction. See method PublishingService.PublishReleaseContentAsync
         // TODO where the Published date on a Release is set in the Content db for the same Live check there.
         public bool Live => Published.HasValue && Compare(UtcNow, Published.Value) > 0;
-        public bool SoftDeleted { get; set; }
```

Comment:
There's no new migration for this, just removal of the existing migration which created it. Is the plan to manually unpick this from the environments that it already reached (so far, only Dev)?

Reply:
Yep it's only made it into dev so far so I will manually delete the field to cut down on number of migrations.